apiary.apib

```diff
@@ -5134,7 +5134,7 @@ Request the upload link for a file.
 + Request (application/json)
 
     + Attributes (object)
-        + name: `meatdealer.png` (string, required)
+        + name: `Important meeting note from August 2022.docx` (string, required) - name under which the file user wants to upload will be stored. This name should contain the same extension as the file user wants to upload.
```

Comment:
PP: The sentence is a bit weird to read.
name under which the file `(that) the` user wants to upload will be stored.

Or just name under which the file will be stored.

Same for the next line
This name should contain the same extension as the file `that the` user wants to upload

Reply:
Fixed: https://github.com/teamleadercrm/api/pull/659/commits/8b1e9bde2c2e11f17ad59b7f2808e09978fbe2cb